dev/tasks/build/start.frag

```diff
@@ -0,0 +1,13 @@
+( function ( root, factory ) {
+	// Register the CKEDITOR global.
+	root.CKEDITOR = factory();
```

Comment:
@Reinmar, it makes sense.

I think we can proceed with the existing code and the do what I already said, to avoid such important thoughts to be left inside commits comments.:

> Ok, I'll open an issue for this once this PR goes accepted.